CONTRIBUTING.md

```diff
@@ -0,0 +1,158 @@
+# Contributing
+
+Thanks for checking out Clay. We accept pull requests from anybody, however we ask
+that you follow some conventions when contributing to Clay.
+
+We have set up a
+[piece-of-cake](https://github.com/pebble/clay/issues?q=is%3Aopen+is%3Aissue+label%3Apiece-of-cake)
+label in GitHub issues for bugs/features that are easy to implement. These issues are
+a great place to start for people that are new to the codebase.
+
+
+## Requirements
+
+ - [Node](https://nodejs.org) (4.0 and later)
+ - [Google Chrome](www.google.com/chrome) (To run tests)
+ - [Pebble SDK](https://developer.pebble.com/sdk/) (optional)
+
+
+## Getting Started
+
+### 1. Fork, then clone the repo:
+
+`$ git clone git@github.com:your-username/clay.git`
+
+### 2. Create a new branch with the following naming convention.
+
+`#<issue-id>/description-of-change`
+
+example:
+
+`#90/support-message-keys`
+
+If there is currently no GitHub issue open for your proposed contribution,
+then please make one. This allows us to assign milestones and triage priority.
+
+### 3. Install the project dependencies.
+
+`$ npm install`
+
+### 4. Develop your feature
+
+Commit as often as you like, however please squash commits that fix typos or are
+otherwise noisy.
+
+### 5. Run the tests
+
+`$ npm run test`
+
+### 6. Run linting
+
+`$ npm run lint`
```

Comment:
They both run as part of CI but are broken up for local development so you can write tests without worrying about small linting issues like extra lines etc.